docs/source/usage_guides/fsdp.md

```diff
@@ -96,6 +96,8 @@ all-gather while executing in the forward pass. only use with Static graphs.
 Useful in cases such as parameter-efficient fine-tuning. 
 Please refer this [blog](https://dev-discuss.pytorch.org/t/rethinking-pytorch-fully-sharded-data-parallel-fsdp-from-first-principles/1019)
 
+`CPU RAM Efficient Model loading`: If True, only the first process loads the pretrained model checkoint while all other processes have empty weights. Only applicable for 🤗 Transformers models. When using this, `Sync Module States` needs to True. 
```

Comment:
Do we know under what circumstances exactly this may cause problems? Maybe it would be worth mentioning, so that those users who currently encounter the problem know that they need to change this argument.

Reply:
Done